presubmit_tests/test-pr-poc.sh

```diff
@@ -46,7 +46,7 @@ fi
 
 # run the install script
 ./install-oracle.sh --ora-swlib-bucket gs://bmaas-testing-oracle-software \
---instance-ssh-user ansible1 --instance-ssh-key /etc/files_needed_for_tk/id_rsa_bms_tk_key \
```

Comment:
I think this was changed unintentionally (and is causing presubmit tests to fail)

Reply:
That file is no longer in the PR.  There had been a permissions change in an earlier commit by another dev, though I do not know why it appeared as changed here.